networking/change-block-size.md

```diff
@@ -0,0 +1,303 @@
+
+### Before you begin...
+
+**Required**
+
+- Verify that you are using {{site.prodname}} IPAM. This guide is only relevant if you are using Calico IPAM.
+
+  ssh to one of your Kubernetes nodes and view the CNI configuration.  
+
+    ```bash
+    cat /etc/cni/net.d/10-calico.conflist
+     ```
+  Look for the "type" entry:
+  <pre>
+     "ipam": {
+           "type": "calico-ipam"
+      }, 
+  </pre>
+
+  If the type is “calico-ipam”, you are good to go. If the IPAM is set to something else, or the 10-calico.conflist file does not exist, you cannot use this feature in your cluster. 
+
+- Verify orchestrator support for changing the pod network CIDR
+
+  Although Kubernetes supports changing the pod network CIDR, not all orchestrators do. For example, **OpenShift** does not support this feature as described in [`osm_cluster_network_cidr configuration`](https://docs.openshift.org/latest/install_config/install/advanced_install.html#configuring-cluster-variables). Check your orchestrator documentation to verify. 
+
+**Recommended**
+
+Understand the basics of [Calico IPAM]({{site.baseurl}}/networking/get-started-ip-addresses)
+
+### How to
+
+>**Important!** Make sure that you follow the steps in the order described to minimize pod connectivity disruption. Pods may lose connectivity when they are redeployed, and may lose external connectivity while in the temporary pool. Also, when pods are deleted, applications may be temporarily unavailable (depending on the type of application). Plan accordingly. 
+{: .alert .alert-danger }
+
+#### Change the IP pool block size
+
+By default, the {{site.prodname}} IPAM block size for an IP pool is /26. To expand from the default size /26, lower the `blockSize` (for example, /8). To shrink the `blockSize` from the default /26, raise the number (for example, /28). 
+
+The high-level steps to follow are:
+
+1. Add a new temporary IP pool.  
+**Note**: The new IP pool must not be within the same cluster CIDR.
+1. Disable the old IP pool.  
+**Note**: Disabling an IP pool prevents only new IP address allocations; it does not affect etworking of existing pods.
+1. Delete pods from the old IP pool.  
+This includes any new pods that may have been created with the old IP pool prior to disabling the pool. Verify that new pods get an address from the new IP pool.
+1. Delete the old IP pool.
+1. Create a new pool with custom CIDR block within the same cluster CIDR.
+1. Disable the temporary IP pool.
+1. Delete pods from the temporary IP pool.
+1. Delete the temporary IP pool
+
+### Tutorial
+
+In the following example, we created a Kubernetes cluster with default CIDR block size of /26. We want to shrink the block size to /28 to use the pool more efficiently.
+
+#### Step 1: Add a new temporary IP pool
+
+We add a new IPPool with the CIDR range, 10.0.0.0/16.
+
+Create a new-pool.yaml.
+
+<pre>
+apiVersion: projectcalico.org/v3
+kind: IPPool
+metadata:
+  name: new-pool
```

Comment:
Maybe name this "temporary-pool" or something to make that more clear? (and change wording in the above high level steps to match)

Reply:
fixed.